src/main/java/eu/lucaventuri/fibry/ActorSystem.java

```diff
@@ -174,7 +176,7 @@ public <T> WeightedActor<T, S> newWeightedPool(Consumer<T> actorLogic, int numPe
                 try (var unlock = lockable.acquire(msg.weight)) {
                     actorLogic.accept(msg.message);
                 } catch (Exception e) {
-                    System.err.println("Weighted actor: " + e);
+                    logger.log(Level.FINEST, "Weighted actor", e);
```

Comment:
Why do you want to hide all these messages?
Please use INFO, not FINEST. In all the lines. Else it would be impossible to notice and debug any issue.